File: 09-palindromo/php/palindromo.php
<?php

$texto = readline("Introduzca una frase: ");
$texto = str_replace(" ", "", $texto);
$texto = strtolower($texto);
$reves = strrev($texto);

if ($texto == $reves) {
    echo "Es palíndromo\n";
} else {
    echo "No es palíndromo\n";
}

File: 09-palindromo/python/palindromo.py
texto = input("Introduzca un texto: ")
texto = texto.lower()
texto = texto.replace(" ", "")
texto_inverso = texto[::-1]
if texto == texto_inverso:
    print("El texto es un palíndromo")
else:
    print("El texto no es un palíndromo")
    